src/shared/components/MapBlock.js

```diff
@@ -39,6 +39,9 @@ class MapBlock extends Component {
       <section style={{ width: '100%', height: '500px' }}>
         <h4>coords: {this.props.center}</h4>
         <GoogleMap
+          bootstrapURLKeys={{
+            key: "AIzaSyDjOUM158Gk2APJ7zfGaRMxgVJ5iMs7M-Q"
```

Comment:
Patrick  said it is not a problem bcz expired within 24hours 
But im still curious